tests/unit/preprocessors/test_converted_notebook_preprocessor.py

```diff
@@ -21,6 +21,14 @@ def test_preprocess():
     os.remove(converted_notebook_path)
     assert Path(converted_notebook_path) in files
 
+def test_not_overwrite_flag():
```

Comment:
Nit: name it "test_not_overwrite_file_for_multiple_runs"?

Reply:
Fixed, thanks.